ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerSessionConfiguration.scala

```diff
@@ -7,13 +7,14 @@ import com.daml.ledger.api.testtool.infrastructure.participant.ParticipantSessio
 import com.daml.ledger.api.tls.TlsConfiguration
 
 private[testtool] final case class LedgerSessionConfiguration(
-    participants: Vector[(String, Int)],
+    private val participantAddresses: Vector[(String, Int)],
+    private val ssl: Option[TlsConfiguration],
+    private val partyAllocation: PartyAllocationConfiguration,
```

Comment:
This definitely does not look as a `case class` any longer. I would recommend you remove the `case` qualifier.

Reply:
Agreed. Done.